### DISCUSSION TITLE
"NativeMagickSettings"

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1245] [Help]
Hi! I foynd discussion about this error, but it didnt help me.

I updated Magick.Net from 8.3 to latest - and any code got this error:
Выполнение действия CSharp OwnCode. Инициализатор типа "NativeMagickSettings" выдал исключение.

This is example of code for this error.

What to do? Help me please!

**Comment by dlemstra** [comment]
On which operating system are you trying to run this code? And which version of .NET are you using?

**Reply by jaka1-newbie** [reply]
Win7 x64

**Reply by jaka1-newbie** [reply]
I dont know how to explain this, but i put near Magick.NET-Q16-HDRI-AnyCPU.dll (12 version) this file Magick.Native-Q16-HDRI-x64.dll and my project has start work!

**Reply by dtechfish** [reply]
Where did you find the Magick.Native dlls ?

**Reply by jaka1-newbie** [reply]
> Where did you find the Magick.Native dlls ?

https://github.com/dlemstra/Magick.NET/discussions/1292
